src/types/index.ts

```diff
@@ -432,6 +444,26 @@ export interface ITopHeader {
 
 export type ISignMessage = (m: any) => Promise<any>
 
+export interface IName {
+  autoExtend: boolean;
+  createdAtHeight: number;
+  expiresAt: number;
+  hash: string;
+  name: string;
+  owner: string;
+  pointers: Record<string, any>;
```

Comment:
I guess we do have a common type `dictionary` for this

Reply:
Sure, makes sense. Updated.